Android/backend/README.md

```diff
@@ -0,0 +1,10 @@
+# Intra Backend
+
+Intra backend is a Go project using Outline SDK.
+
+## Build
+
+```sh
+gomobile clean
```

Comment:
It's not clear how to set up go mobile.
We need to specify it in tools.go to make sure the version is fixed.
You shouldn't use the system binary.